### PR TITLE
Match type should not always be ISBN

### PR DESCRIPTION
**Pull request by cableman**
https://jira.itkdev.dk/browse/DDBTEAM-462

### CODE REVIEW

File: tests/UploadServiceVendorServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    /**
+     * Test extraction of filename.
+     *
+     * @throws \ReflectionException
+     */
+    public function testExtractFilename()
+    {
+        $service = $this->getUploadServiceVendorService();
+
+        // Have to use full namespace in reaction class or it will fail.
```
Comment (tuj) [inline tests/UploadServiceVendorServiceTest.php:33]: reaction => reflection

File: tests/UploadServiceVendorServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+     * Test filename to identifier.
+     *
+     * @throws \ReflectionException
+     */
+    public function testFilenameToIdentifier()
+    {
+        $service = $this->getUploadServiceVendorService();
+
+        // Have to use full namespace in reaction class or it will fail.
+        $ref = new \ReflectionClass('\App\Service\VendorService\UploadService\UploadServiceVendorService');
```
Comment (tuj) [inline tests/UploadServiceVendorServiceTest.php:52]: reaction => reflection

File: tests/UploadServiceVendorServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        $method->setAccessible(true);
+
+        $this->assertEquals($method->invoke($service, '870970-basis:06390072'), IdentifierType::PID);
+        $this->assertEquals($method->invoke($service, '9788702284799'), IdentifierType::ISBN);
+    }
+
+    /**
+     * Helper class to mock service for UploadService.
+     *
+     * @return uploadServiceVendorService
```
Comment (tuj) [inline tests/UploadServiceVendorServiceTest.php:82]: uploadServiceVendorService => UploadServiceVendorService

File: tests/UploadServiceVendorServiceTest.php (hunk 1 above)
Comment (cableman) [reply tests/UploadServiceVendorServiceTest.php:33]: Fixed

File: tests/UploadServiceVendorServiceTest.php (hunk 2 above)
Comment (cableman) [reply tests/UploadServiceVendorServiceTest.php:52]: Fixed

File: tests/UploadServiceVendorServiceTest.php (hunk 3 above)
Comment (cableman) [reply tests/UploadServiceVendorServiceTest.php:82]: Fixed

**Pull request merged**